libs/sqlserver/langchain_sqlserver/vectorstores.py

```diff
@@ -215,21 +218,38 @@ def __init__(
                 Optional param, defaults to None.
             table_name: The name of the table to use for storing embeddings.
                 Default value is `sqlserver_vectorstore`.
+            batch_size: Number of documents/texts to be inserted at once to Db, max 419.
+
         """
+        batch_size = self._validate_batch_size(batch_size)
         self.connection_string = self._get_connection_url(connection_string)
         self._distance_strategy = distance_strategy
         self.embedding_function = embedding_function
         self._embedding_length = embedding_length
         self.schema = db_schema
         self.override_relevance_score_fn = relevance_score_fn
         self.table_name = table_name
+        self.__batch_size = batch_size
```

Comment:
nit: for consistency should `__batch_size` be `_batch_size`. Python docs say double leading and trailing underscores are reserved for special use in the language. This is supposed to be a private variable.